src/langs/assemblyscript/wheel-part.ts

```diff
@@ -1,7 +1,10 @@
+@external("env", "random")
+declare function random(): f64;
+
 export function name(): string {
   return "AssemblyScript";
 }
 
-export function feelingLucky(): number {
-  return Math.floor(Math.random() * 100) + 1;
-}
\ No newline at end of file
+export function feelingLucky(): f64 {
```

Comment:
The idea is that `feelingLucky` should return a random number between 1 and 100, so maybe a better type would be `i64`.

Reply:
Unfortunately interop between js <-> wasm can't accept 64-bit integers. btw even if you return `f32` or `i32` internally this types always cast to `f64`/`number`.